t/app/controller/admin/users.t

```diff
@@ -550,6 +550,26 @@ FixMyStreet::override_config {
         $mech->content_contains( 'Updated!' );
     };
 
+    subtest "Test edit user name who has verified landline" => sub {
+        my $existing_user = $mech->create_user_ok('existing@example.com', name => 'Existing User', phone => '01184960017', phone_verified => 1 );
+	$mech->get_ok( '/admin/users/' . $existing_user->id );
+	$mech->submit_form_ok( { with_fields => {
+            name => 'Existing E. User',
+				 } } );
+	$mech->content_contains( 'Updated!' );
+    };
+
+    subtest "Test edit user number who has verified landline" => sub {
+        my $existing_user = $mech->create_user_ok('existing@example.com', name => 'Existing User', phone => '01184960017', phone_verified => 1 );
+	$mech->get_ok( '/admin/users/' . $existing_user->id );
+	$mech->submit_form_ok( { with_fields => {
+            phone => '01184960018',
+	} } );
+	$mech->content_lacks( 'Updated!' );
```

Comment:
Would be nice to check for the expected error message here too.

Reply:
Done